wafer/settings.py

```diff
@@ -148,6 +148,8 @@
     'wafer.sponsors',
     'wafer.pages',
     'wafer.tickets',
+    # Django isn't finding the overridden templates
+    'registration',
```

Comment:
This might've just been an issue with the version of Django I was using, but there was an issue where Django was only finding the `registration` library's templates, and not our custom versions it should use.